data_steward/tools/participant_prevalence.py

```diff
@@ -0,0 +1,244 @@
+    return non_zero_counts
+
+
+def estimate_prevalence(project_id, hpo_id, pids_string):
+    """
+    Logs dataset_name, table_name, non_cdm_count and ehr_count to count rows pertaining to pids
+
+    :param project_id: identifies the project
+    :param hpo_id: Identifies the hpo site that submitted the pids
+    :param pids_string: string containing query or pids in bq string format
+    :return:
+    """
+    datasets = bq.list_datasets(project_id)
+    for dataset in datasets:
+        dataset_id = dataset.dataset_id
+        try:
+            non_zero_counts_df = get_non_zero_counts(project_id, dataset_id,
+                                                     hpo_id, pids_string)
+            non_zero_counts = non_zero_counts_df.get_values()
+
+            if non_zero_counts.size > 0:
+                for count_row in non_zero_counts:
+                    logging.info(
+                        'DATASET_ID: {}\tTABLE_ID: {}\tALL_COUNT: {}\tEHR_COUNT: {}'
+                        .format(dataset_id, *count_row))
+        except BadRequest:
+            logging.exception(f'Dataset {dataset_id} could not be analyzed')
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Estimates the prevalence of specified pids in the project',
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('-p',
+                        '--project_id',
+                        action='store',
+                        dest='project_id',
+                        help='Identifies the project to retract data from',
+                        required=True)
+    parser.add_argument('-o',
+                        '--hpo_id',
+                        action='store',
+                        dest='hpo_id',
+                        help='Identifies the site submitting the person_ids',
+                        required=True)
+    parser.add_argument(
+        '-q',
+        '--pid_project_id',
+        action='store',
+        dest='pid_project_id',
+        help='Identifies the project containing the sandbox dataset',
+        required=False)
+    parser.add_argument('-s',
+                        '--sandbox_dataset_id',
+                        action='store',
+                        dest='sandbox_dataset_id',
+                        help='Identifies the dataset containing the pid table',
+                        required=False)
+    parser.add_argument('-t',
+                        '--pid_table_id',
+                        action='store',
+                        dest='pid_table_id',
+                        help='Identifies the table containing the person_ids',
+                        required=False)
+    parser.add_argument('-i',
+                        '--pid_list',
+                        nargs='+',
+                        type=int,
+                        dest='pid_list',
+                        help='Person_ids to check for',
+                        required=False)
```

Comment:
@nishanthpp93 please research on subparser and let me know what you find. We determined that these don't make sense:
```
participant_prevalence --project_id my_project --hpo_id my_hpo --pid_source table pid_table

participant_prevalence --project_id my_project --hpo_id my_hpo --pid_source table pid_dataset.pid_table
```
So if did allow qualified table_id this would have to be the **only** way we accept it. We would have to do a regex to validate the input. Just a suggestion- it's up to you.